src/_scss/pages/recipientLanding/recipientLandingPage.scss

```diff
@@ -0,0 +1,81 @@
+.usa-da-recipient-landing {
+    @import "all";
+    @import "layouts/landingPage/landingPage";
+    @import "layouts/default/stickyHeader/header";
+    @import "pages/accountLanding/searchSection";
+    @import "components/pagination";
+    @import "./_table";
+    @import "./_recipientLabels";
+
+    .pagination {
+        .pager {
+            justify-content: flex-end;
+        }
+    }
+
+    .recipient-landing__results {
+        .recipient-list {
+            margin-top: rem(15);
+        }
+        .recipient-list__message {
+            margin: rem(90) auto;
+            font-size: rem(36);
+            font-weight: 300;
+            color: $color-base;
+            text-align: center;
+            .recipient-list__message_highlight {
+                color: $color-active;
+                font-weight: normal;
+            }
+        }
+    }
+
+    .recipient-landing__icon-parent {
```

Comment:
These labels can have one class for the shared styling (`recipient-landing__icon`) and a modifier class `&.recipient-landing__icon_parent` that specifies the background color